docs/install/run.md

```diff
@@ -447,6 +446,308 @@ curl -k -XPOST https://127.0.0.1:5000/api/records/jnmmp-51n47/draft/actions/publ
+            "website": "https://shea.org"
+        },
+        "links": {
+            "self": "https://127.0.0.1:5000/api/communities/foster-anderson",
+            "self_html": "https://127.0.0.1:5000/communities/foster-anderson",
+            "settings_html": "https://127.0.0.1:5000/communities/foster-anderson/settings",
+            "logo": "https://127.0.0.1:5000/api/communities/foster-anderson/logo",
+            "rename": "https://127.0.0.1:5000/api/communities/foster-anderson/rename"
+        },
+        "created": "2021-04-29T13:07:06.996160+00:00",
+        "access": {
+            "member_policy": "closed",
+            "owned_by": [],
+            "visibility": "public",
+            "record_policy": "open"
+        },
+        "updated": "2021-04-29T13:07:07.014840+00:00"
+        }
+    ],
+    "total": 65
+    },
+    "sortBy": "newest",
+    "aggregations": {
+    "domain": {
+        "doc_count_error_upper_bound": 0,
+        "sum_other_doc_count": 0,
+        "buckets": []
+    },
+    "type": {
+        "doc_count_error_upper_bound": 0,
+        "sum_other_doc_count": 0,
+        "buckets": [
+        {
+            "key": "organization",
+            "doc_count": 21
+        },
+        {
+            "key": "event",
+            "doc_count": 20
+        },
+        {
+            "key": "topic",
+            "doc_count": 13
+        },
+        {
+            "key": "project",
+            "doc_count": 10
+        }
+        ]
+      }
+    },
+    "links": {
+    "self": "https://127.0.0.1:5000/api/communities?page=1&size=25&sort=newest",
+    "next": "https://127.0.0.1:5000/api/communities?page=2&size=25&sort=newest"
+    }
+  }
+}
+```
+
+#### Notes
+- Results are abbreviated for ease of reading.
+- [jq](https://github.com/stedolan/jq) is for pretty formating and coloring.
+
+
+### Create communities
+
+!!! info "Authentication required"
+
+    All requests to the create-related REST API endpoints require authentication.
+#### Create a community
+
+You can create a new community using the API:
+
+```bash
+curl -k -XPOST -H "Content-Type: application/json" https://127.0.0.1:5000/api/communities?access_token=<token> -d '{
+  "access": {
+    "visibility": "public",
+    "member_policy": "open",
+    "record_policy": "open"
+  },
+  "id": "yet_another_community",
+  "metadata": {
+    "title": "Brand New Community",
+    "description": "This is an example Community.",
+    "type": "event",
+    "curation_policy": "This is the kind of records we accept.",
+    "page": "Information for my community.",
+    "website": "https://inveniosoftware.org/"
```

Comment:
You can use a real example here, e.g. "Awesome Repositories Conf 2021", with an appropriate description/curation_policy, etc.